src_new/bidManager.js

```diff
@@ -389,9 +389,9 @@ function analyticalPixelCallback(slotID, bmEntry, impressionIDMap) { // TDD, i/o
                 return;
             }
 
-						if (adapterID === "pubmaticServer") {
-								return;
-						}
+						// if (adapterID === "pubmaticServer") {
```

Comment:
I think block should not be commented.
This block was added to not pass data for pubmaticServer adapter into logger as it is not actually an adapter but a container of server-side adapters.